serverless/knative_eventing/serverless-understanding-eventing.adoc

```diff
@@ -1,36 +1,42 @@
+== Knative Eventing use cases
 
-Events are buffered if the destination sink is unavailable.
-// For more information about creating an Event delivery system, see
-// Add link to getting started or tutorial landing page for eventing
-Knative Eventing supports the following scenarios:
+Knative Eventing supports the following use cases:
 
-Publish an event without creating a consumer:: You can send events to a broker as an HTTP POST, and use a SinkBinding to decouple the destination configuration from your application that is producing events.
+Publish an event without creating a consumer:: You can send events to a broker as an HTTP POST, and use binding to decouple the destination configuration from your application that is producing events.
 Consume an event without creating a publisher:: You can use a trigger to consume events from a broker based on event attributes. Your application will receive events as an HTTP POST.
+
 // Use Channels and Subscriptions to define complex message-passing topologies:: For simple pipelines, the Sequence automates construction of Channels and Subscriptions between each stage.
 // Knative also supports some additional patterns such as Parallel fanout of events, and routing response events from both Channels and Brokers
 
-[id="serverless-event-architecture-sinks"]
-== Event sinks
+// Add stuff about event delivery
+
+[id="serverless-understanding-eventing-objects"]
+== Knative Eventing objects
 
 To enable delivery to multiple types of sinks, Knative Eventing defines the following generic interfaces that can be implemented by multiple Kubernetes resources:
 
-Addressable objects:: Able to receive and acknowledge an event delivered over HTTP to an address defined in the event's `status.address.url` field. The Kubernetes Service object also satisfies the addressable interface.
-Callable objects:: Able to receive an event delivered over HTTP and transform it, returning 0 or 1 new events in the HTTP response payload. These returned events may be further processed in the same way that events from an external event source are processed.
+Addressable objects:: Able to receive and acknowledge an event delivered over HTTP to an address defined in the event's `status.address.url` field. The Kubernetes `Service` object also satisfies the addressable interface.
+Callable objects:: Able to receive an event delivered over HTTP and transform it, returning `0` or `1` new events in the HTTP response payload. These returned events may be further processed in the same way that events from an external event source are processed.
+
+You can propagate an event from an xref:../../serverless/event_sources/knative-event-sources.adoc#knative-event-sources[event source] to multiple event sinks by using:
+
+* xref:../../serverless/channels/serverless-channels.adoc#serverless-channels[channels] and subscriptions, or
+* xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers[brokers] and triggers.
```

Comment:
Should _triggers_ be linked to https://deploy-preview-33595--osdocs.netlify.app/openshift-enterprise/latest/serverless/knative_eventing/serverless-triggers.html ??